docs/Makefile

```diff
@@ -20,11 +20,11 @@ docset: html
 	convert $(SPHINXPROJ).docset/icon@2x.png -resize 16x16 $(SPHINXPROJ).docset/icon.png
 
 rebuild:
-	rm -rf source/generated && make clean && make html
+	rm -rf source/generated && find ../ignite -name *.pyc -delete && rm -rf source/__pycache__ && make clean && make html
```

Comment:
Why do we need to update this command ?

Reply:
In a local environment (and not in CI) in which the developer might change ignite docstrings and see the effect after build over and over, the changes didn't take effect unless one removes the `.pyc` files in the ignite pycache.